### ISSUE TITLE
Job Scheduler no logged Job execution - Workers grid is empty

### ISSUE THREAD

**Issue by ekuziel** [eXpandFramework/eXpand#992]
𝗤𝘂𝗲𝘀𝘁𝗶𝗼𝗻
Hi, I registered Reactive.XAF Job Scheduler module with Hangfire in application. After run of registered Job - the action has been executed succesfully and working as expected, but no data in **workers** grid occured, data in Hangfire table are populated but not in dbo.JobState and dbo.JobWorker.

Am I missing some configuration related to that XAF module ? Do you have some sample project with Reactive XAF modules ?

Hangfire configuration:
`      services.AddHangfire(configuration => configuration
            .UseSimpleAssemblyNameTypeSerializer()
            .UseRecommendedSerializerSettings()
            .UseSqlServerStorage(connectionString)
        );`

Ractive XAF version 22.5, DevExpress XAF 22.5

**Comment by apobekiaris** [comment]
in general u do not need to configure hangfire to default as you already used this (i guess)
```
[assembly: HostingStartup(typeof(HangfireStartup))]
 [assembly: HostingStartup(typeof(HostingStartup))]
 [assembly:HostingStartup(typeof(Xpand.XAF.Modules.Blazor.BlazorStartup))]
```

they configure hangfire, the rest of the integration is done from Xaf so you need the JobScheduler module as well in the RequiredModuleTypes collection

**Comment by apobekiaris** [comment]
I am now working on another JobScheduler issue so I can verify that workers are created with this sample
https://github.com/eXpandFramework/eXpand/issues/988#issuecomment-1516045038

can u test it ?

**Comment by ekuziel** [comment]
hi, validated source from #988 -> hangfire schema wasn't created after first or second run (set to SQL server db instead localdb), - even when I triggered Job - I can see logged message in WOrkers grid - but can see details and also can't see hangfire Dashboard - I think due to hangfire schema doesn't exist - I will look on it later - but maybe creating of missing schema needs to be solved

**Comment by apobekiaris** [comment]
looks like the view is configured for MasterDetailMode = ListViewAndDetailView u can change it on your side until there is a fix

**Comment by apobekiaris** [comment]
this case is also fixed in the next minor

**Comment by expand** [comment]
The pre-release [4.222.11.0](https://github.com/eXpandFramework/Reactive.XAF/releases/tag/4.222.11.0) in the [Reactive.XAF](https://github.com/eXpandFramework/Reactive.XAF/tree/lab) `lab` branch  includes commits that relate to this task:

* [Fix Hangfire view issues #992](https://github.com/eXpandFramework/Reactive.XAF/commit/413fd13eeca8eee6000163de131e3cff57ad7299)

To minimize version conflicts we recommend that you use the [Xpand.XAF.Core.All](https://www.nuget.org/packages/Xpand.XAF.Core.All), [Xpand.XAF.Win.All](https://www.nuget.org/packages/Xpand.XAF.Win.All), [Xpand.XAF.Web.All](https://www.nuget.org/packages/Xpand.XAF.Web.All) packages. Doing so, all packages will be at your disposal and .NET will add a dependecy only to those packages that you actually use and not to all (see the [Modules installation-registrations](https://youtu.be/LvxQ-U_0Sbg) youtube video).
<details>
    <summary>Released packages:</summary>
<a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions'>Xpand.Extensions v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Blazor'>Xpand.Extensions.Blazor v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Mono.Cecil'>Xpand.Extensions.Mono.Cecil v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud'>Xpand.Extensions.Office.Cloud v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Office.Cloud.Google.Blazor'>Xpand.Extensions.Office.Cloud.Google.Blazor v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.Reactive'>Xpand.Extensions.Reactive v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF'>Xpand.Extensions.XAF v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Extensions/Xpand.Extensions.XAF.Xpo'>Xpand.Extensions.XAF.Xpo v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Blazor v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.Common v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/tests'>Xpand.TestsLib.EasyTest v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/tools/Xpand.VersionConverter'>Xpand.VersionConverter v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Core.All v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/AutoCommit'>Xpand.XAF.Modules.AutoCommit v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Blazor'>Xpand.XAF.Modules.Blazor v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/BulkObjectUpdate'>Xpand.XAF.Modules.BulkObjectUpdate v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneMemberValue'>Xpand.XAF.Modules.CloneMemberValue v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/CloneModelView'>Xpand.XAF.Modules.CloneModelView v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Email'>Xpand.XAF.Modules.Email v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/GridListEditor'>Xpand.XAF.Modules.GridListEditor v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/HideToolBar'>Xpand.XAF.Modules.HideToolBar v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Hangfire'>Xpand.XAF.Modules.JobScheduler.Hangfire v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/JobScheduler.Notification'>Xpand.XAF.Modules.JobScheduler.Notification v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/MasterDetail'>Xpand.XAF.Modules.MasterDetail v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelMapper'>Xpand.XAF.Modules.ModelMapper v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ModelViewInheritance'>Xpand.XAF.Modules.ModelViewInheritance v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google'>Xpand.XAF.Modules.Office.Cloud.Google v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Calendar'>Xpand.XAF.Modules.Office.Cloud.Google.Calendar v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.Cloud.Google.Tasks'>Xpand.XAF.Modules.Office.Cloud.Google.Tasks v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Office.DocumentStyleManager'>Xpand.XAF.Modules.Office.DocumentStyleManager v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/OneView'>Xpand.XAF.Modules.OneView v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/PositionInListView'>Xpand.XAF.Modules.PositionInListView v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ProgressBarViewItem'>Xpand.XAF.Modules.ProgressBarViewItem v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RazorView'>Xpand.XAF.Modules.RazorView v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive'>Xpand.XAF.Modules.Reactive v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger'>Xpand.XAF.Modules.Reactive.Logger v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Client.Win'>Xpand.XAF.Modules.Reactive.Logger.Client.Win v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Logger.Hub'>Xpand.XAF.Modules.Reactive.Logger.Hub v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Reactive.Rest'>Xpand.XAF.Modules.Reactive.Rest v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/RefreshView'>Xpand.XAF.Modules.RefreshView v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SequenceGenerator'>Xpand.XAF.Modules.SequenceGenerator v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Speech'>Xpand.XAF.Modules.Speech v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SpellChecker'>Xpand.XAF.Modules.SpellChecker v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/StoreToDisk'>Xpand.XAF.Modules.StoreToDisk v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/SuppressConfirmation'>Xpand.XAF.Modules.SuppressConfirmation v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/TenantManager'>Xpand.XAF.Modules.TenantManager v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewEditMode'>Xpand.XAF.Modules.ViewEditMode v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewItemValue'>Xpand.XAF.Modules.ViewItemValue v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/ViewWizard'>Xpand.XAF.Modules.ViewWizard v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules/Windows'>Xpand.XAF.Modules.Windows v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Web.All v.4.222.13</a></br> <a href='https://github.com/eXpandFramework/Reactive.XAF/tree/master/src/Modules'>Xpand.XAF.Win.All v.4.222.13</a></br>
</details>

Please update the related Nuget packages and test if issues is addressed. These are nightly nuget packages available only from our [NugetServer](https://xpandnugetserver.azurewebsites.net/nuget/).

If you do not use these packages directly but through a module of the main eXpandFramework project, please wait for the bot to notify you again when integration is finished or update the related packages manually.

Thanks a lot for your contribution.

**Comment by expand** [comment]
Closing issue for age. Feel free to reopen it at any time.

.Thank you for your contribution.
